capstone/src/main/java/com/google/sps/filter/AuthenticateFilter.java

```diff
@@ -33,7 +33,7 @@
     private static final String AUTH_HEADER_NAME = "id_token";
 
     public void init(FilterConfig filterConfig) throws ServletException{
-
+        
```

Comment:
Minor thing so not even worth the trouble of updating the PR, but its preferable to group whitespace/formatting updates in their own PR full of meaningless changes so that it is very easy to review and just "yup yup yup, approve". When it is part of content PRs it clutters up the diff

Reply:
Got it. Makes sense.

Reply:
Makes sense. It will feel nice once we have our project done and just removing white space to make it cleaner :D.